src/DotNetWorker.Core/Hosting/WorkerOptions.cs

```diff
@@ -21,5 +22,11 @@ public class WorkerOptions
         /// Gets the collection of input converters.
         /// </summary>
         public InputConverterCollection InputConverters { get; } = new InputConverterCollection();
+
+        /// <summary>
+        /// Gets and sets the flag for opting in to surfacing user-code-thrown
+        /// exceptions to app insights. 
+        /// </summary>
+        public bool EnableUserCodeException { get; set; } = false;
```

Comment:
QQ: who is setting this value? Wondering if we meant to use the feature flag via worker.config like we do for worker indexing?

https://github.com/Azure/azure-functions-host/blob/eb8b9dffdc5a417c886c290033cadde68b403ed4/src/WebJobs.Script/Utility.cs#L913-L934

Reply:
From my understanding, this is meant to be set by the customer - presenting the opt-in option to them since there is a possibility that certain customers may have dependencies on/subscribe to specific errors which would change with enabling this feature.

Reply:
@fabiocav , for other language worker that don't have a bootstrap process like java, is it a good way that we use appsetting to allow cx enable this feature flag? Thanks.

Reply:
You should be able to use app settings